_maps/map_files/stations/boxstation.dmm

```diff
@@ -656,7 +656,6 @@
 /obj/machinery/requests_console{
 	department = "Security";
 	departmentType = 5;
-	name = "Security Requests Console";
 	pixel_y = 30
```

Comment:
these are still so snowflaked. we can subtype these, we have the technology!

Reply:
WYCI

Reply:
true!